base/applications/regedit/listview.c

```diff
@@ -24,8 +11,8 @@
 #define CY_ICON            16
 #define LISTVIEW_NUM_ICONS 2
 
-int Image_String = 0;
-int Image_Bin = 0;
+int Image_String;
+int Image_Bin;
```

Comment:
Not sure this fixes anything.

Reply:
This doesn't fix anything. But is shorter.

Reply:
Doesn't make the code that much correct either.